### PR TITLE
R-0.0.6

### PR DESCRIPTION
**Pull request by bob-crutchley**

### CODE REVIEW

File: topics/nginx/modules/http-reverse-proxy-configuration/README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+- [Overview](#overview)
+- [Reasons for Using a Reverse Proxy](#reasons-for-using-a-reverse-proxy)
+	- [Load Balancing & Global Server Load Balancing](#load-balancing--global-server-load-balancing)
+	- [Protection from Attacks](#protection-from-attacks)
+	- [Caching](#caching)
+	- [TLS/SSL Encryption](#tlsssl-encryption)
+	- [Visual Representation](#visual-representation)
+- [Configuration](#configuration)
+	- [Basic](#basic)
+	- [Multi-Tier Applications](#multitier-applications)
```
Comment (DevQAC) [inline topics/nginx/modules/http-reverse-proxy-configuration/README.md]: Has this been created yet? The link is broken if so. If not, no problem!

File: topics/nginx/modules/http-reverse-proxy-configuration/README.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    server {
+        listen 80;
+        location / {
+            proxy_pass http://web-app;
+        }
+    }
+}
+```
+### Multi-Tier Applications
+Applications are often developed in multiple tiers, frontend, backend and database for example.
```
Comment (DevQAC) [inline topics/nginx/modules/http-reverse-proxy-configuration/README.md]:
```suggestion
Applications are often developed in multiple tiers; front-end, back-end and database for example.
```

File: topics/powershell/modules/functions/README.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+- [Task](#task)
+
+<!--TOC_END-->
+## Overview
+
+Instead of copying and pasting the same code over and over again, we can use a function. We will be learning about functions in Powershell in this module, but the concepts also translate across all programming languages.
+
+## Powershell Functions
+
+When you first start out with Powershell, you aren't always too concerned with things such as modularity, reusibility and "best practices". As time goes on, you will very quickly realise that you are repeating yourself in code, a lot.
```
Comment (DevQAC) [inline topics/powershell/modules/functions/README.md]:
```suggestion
When you first start out with Powershell, you aren't always too concerned with things such as modularity, reusibility and "best practices". As time goes on, you will very quickly realise that you are repeating yourself in code often.
```

File: topics/nginx/modules/http-reverse-proxy-configuration/README.md (hunk 1 above)
Comment (bob-crutchley) [reply topics/nginx/modules/http-reverse-proxy-configuration/README.md]: Ah this is a bug with the table of contents generator, I've raised an issue for this: #378

**Pull request merged**